test/Microsoft.TestPlatform.PerformanceTests/ProtocolV1Tests.cs

```diff
@@ -30,97 +32,96 @@ public class ProtocolV1Tests
         ErrorStackTrace = "sampleStackTrace",
         DisplayName = "sampleTestResult",
         ComputerName = "sampleComputerName",
         Duration = TimeSpan.MaxValue,
-        StartTime = s_startTime,
+        StartTime = StartTime,
         EndTime = DateTimeOffset.MaxValue
     };
 
     [TestMethod]
     public void TestCaseSerialize()
     {
-        Serialize(TestCase);
+        SerializeV1(TestCase);
         Stopwatch sw = Stopwatch.StartNew();
 
         for (int i = 0; i < 10000; i++)
         {
-            Serialize(TestCase);
+            SerializeV1(TestCase);
         }
         sw.Stop();
 
-        VerifyPerformanceResult("TestCaseSerialize1", 2000, sw.ElapsedMilliseconds);
+        var actualDuration = sw.Elapsed;
+        actualDuration.Should().BeLessOrEqualTo(2.Seconds(), $"when serializing 10k test cases");
     }
 
     [TestMethod]
     public void TestCaseDeserialize()
     {
-        var json = Serialize(TestCase);
-        Deserialize<TestCase>(json);
+        var json = SerializeV1(TestCase);
+        DeserializeV1<TestCase>(json);
 
         Stopwatch sw = Stopwatch.StartNew();
         for (int i = 0; i < 10000; i++)
         {
-            Deserialize<TestCase>(json);
+            DeserializeV1<TestCase>(json);
         }
         sw.Stop();
 
-        VerifyPerformanceResult("TestCaseDeserialize1", 2000, sw.ElapsedMilliseconds);
+        var actualDuration = sw.Elapsed;
+        actualDuration.Should().BeLessOrEqualTo(2.Seconds(), $"when de-serializing 10k test cases");
     }
 
     [TestMethod]
     public void TestResultSerialize()
     {
-        Serialize(TestResult);
+        SerializeV1(TestResult);
         Stopwatch sw = Stopwatch.StartNew();
 
         for (int i = 0; i < 10000; i++)
         {
-            Serialize(TestResult);
+            SerializeV1(TestResult);
         }
         sw.Stop();
 
-        VerifyPerformanceResult("TestResultSerialize1", 2000, sw.ElapsedMilliseconds);
+        var actualDuration = sw.Elapsed;
+        actualDuration.Should().BeLessOrEqualTo(2.Seconds(), $"when serializing 10k test results");
     }
 
     [TestMethod]
     public void TestResultDeserialize()
     {
-        var json = Serialize(TestResult);
-        Deserialize<TestResult>(json);
+        var json = SerializeV1(TestResult);
+        DeserializeV1<TestResult>(json);
 
         Stopwatch sw = Stopwatch.StartNew();
         for (int i = 0; i < 10000; i++)
         {
-            Deserialize<TestResult>(json);
+            DeserializeV1<TestResult>(json);
         }
         sw.Stop();
 
-        VerifyPerformanceResult("TestResultDeserialize1", 3500, sw.ElapsedMilliseconds);
-    }
-
-    private static string Serialize<T>(T data, int version = 1)
-    {
-        return JsonDataSerializer.Instance.Serialize(data, version);
+        var actualDuration = sw.Elapsed;
+        actualDuration.Should().BeLessOrEqualTo(3.5.Seconds(), $"when de-serializing 10k test results");
```

Comment:
Nit: interpolation is not needed.
```suggestion
        actualDuration.Should().BeLessOrEqualTo(3.5.Seconds(), "when de-serializing 10k test results");
```